data/speakers.yml

```diff
@@ -10937,13 +10923,6 @@
   website: ablogaboutcode.com
   slug: pan-thomakos
   canonical_slug:
-- name: Panayiotis Thomakos
-  twitter: ""
-  github: ""
-  bio: ""
-  website: ""
-  slug: panayiotis-thomakos
-  canonical_slug:
```

Comment:
Let's keep this one too and set the `canonical_slug` to `pan-thomakos` so future references point to the right record.